vendor/vendor.json

```diff
@@ -1222,6 +1222,12 @@
 			"revision": "653207bc29a6d2d62b5d4f55b596467cb715a128",
 			"revisionTime": "2017-03-27T18:58:03Z"
 		},
+		{
+			"checksumSHA1": "kZRhErakejBG0U2e8D+Ap/Djje8=",
```

Comment:
I assume you will need to run `make update` on the top level so the NOTICE.txt file is updated.

Reply:
Yes! Done in latest push